flutter/rsvp/lib/ViewModels/CreateEventViewModel.dart

```diff
@@ -0,0 +1,8 @@
+class CreateEventViewModel {
+  String defaultCreatorName = "Unknown";
+
+  createEventViewModel(String eventName, String eventDate, int minNum, String eventDescription, String eventLocation) {
```

Comment:
These shouldn't be constructor parameters, there should be a method in this class called `createEvent()` that has these parameters and is called from the view.

Reply:
If you see closely, that is NOT a constructor. I made you confused. I will change the name of the method.